lapis2/src/main/kotlin/org/genspectrum/lapis/controller/ExceptionHandler.kt

```diff
@@ -70,18 +70,17 @@ class ExceptionHandler : ResponseEntityExceptionHandler() {
     }
 
     @ExceptionHandler(SiloException::class)
-    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
     fun handleSiloException(e: SiloException): ErrorResponse {
-        log.error(e) { "Caught SiloException: ${e.message}" }
+        log.error(e) { "Caught SiloException: ${e.statusCode} - ${e.message}" }
 
         return ResponseEntity
-            .status(HttpStatus.INTERNAL_SERVER_ERROR)
+            .status(e.statusCode)
```

Comment:
Probably not. Say SILO returns a 400, then we know that someone messed up the SILO query. 
* It might be that we didn't properly assemble it from the user's data.
* We don't do much validation (intentionally) in LAPIS, so it might also be that the user specified a wrong field name or segment name.

In the first case, 500 would be correct. In the second case, forwarding the 400 from SILO is correct. Since we don't validate ourselves in LAPIS, I think the second case is more relevant here and will (hopefully) more often return the correct status.